tests/testthat/test-backend-mssql.R

```diff
@@ -400,6 +400,12 @@ test_that("row_number() with and without group_by() and arrange(): unordered def
   expect_snapshot(mf %>% arrange(y) %>% mutate(rown = row_number()))
 })
 
+test_that("count_big", {
+  mf <- lazy_frame(x = c(1:5), y = c(rep("A", 5)), con = simulate_mssql())
+  expect_snapshot(count(mf))
+  expect_snapshot(tally(mf))
```

Comment:
I think one snapshot test is adequate here

Reply:
Makes sense. Have removed the second one